src/global.scss

```diff
@@ -180,13 +180,27 @@ select {
   display: block;
   position: relative;
   right: 30px;
+  font-size: 16px;
 }
 
 .searchbar-search-icon.sc-ion-searchbar-ios.ios.hydrated {
   position: relative;
-  left: -14px;
+  left: -102%;
```

Comment:
are you sure this is working fine and is responsive?

Reply:
i made it and i test it
![Screenshot 2023-12-02 1](https://github.com/Getsov/Motorsport-Events-App/assets/114027141/a13816fc-0e55-480c-a4b4-9717a9b34143)
![Screenshot 2023-12-02 2](https://github.com/Getsov/Motorsport-Events-App/assets/114027141/93fb3674-78e5-46c7-bc6d-bf4fd36a5f1b)
![Screenshot 2023-12-02 3](https://github.com/Getsov/Motorsport-Events-App/assets/114027141/2adf8bad-d5d9-44d6-ac16-24f1bf9c4b38)